parlai/tasks/multiwoz_v22/agents.py

```diff
@@ -373,6 +379,212 @@ def get_id_task_prefix(self):
         return "MultiwozV22"
 
 
+class MultiWOZv22DSTTeacher(tod_agents.TodUserSimulatorTeacher):
```

Comment:
might be better to add a comment saying this teacher is needed for reproducing the Joint Goal Accuracy values reported in the simpleTOD & SOLOIST papers.

Reply:
Is it better now ? @chinnadhurai